lib/define/mysql.js

```diff
@@ -187,7 +191,7 @@ MysqlConnection.prototype.insert = function(table, sets, callback) {
     })
     return
   }
-  assert(sets) 
+  assert(sets)
```

Comment:
change 'assert' to callback(Error)

otherwise, user need both 'try' and handle callback Error